website/content/en/applications.adoc

```diff
@@ -3,41 +3,57 @@ title: "About Applications for FreeBSD"
 sidenav: about
 ---
 
-= About Applications for FreeBSD
+= Applications for FreeBSD
 
-== Experience the possibilities with FreeBSD
+FreeBSD goes beyond what's commonly expected of a UNIX(R) workstation.
 
-FreeBSD can handle nearly any task you would expect of a UNIX(R) workstation, as well as many you might not expect:
+== Thousands of applications
 
-== FreeBSD is a true open system with full source code.
+There are more than thirty thousand link:../ports/[ports] and related packages.
 
-There is no doubt that so-called open systems are _the_ requirement for today's computing applications. But no commercial vendor-supplied solution is more open than one which includes full source code to the entire operating system, including the kernel and all of the system daemons, programs, and utilities. You can modify any part of FreeBSD to suit your personal, organizational, or corporate needs.
+Non-ported software will often compile and run with ease.
+FreeBSD has https://freebsdfoundation.org/freebsd/timeline/[UNIX and BSD heritage].
 
-With its generous link:../copyright/freebsd-license/[licensing policy], you can use FreeBSD as the basis for any number of free _or commercial_ applications.
+== Use cases
 
-== FreeBSD runs thousands of applications.
+*Internet services*.
+FreeBSD is famed for performant, reliable networking for Internet service providers.
+Ported software includes https://nginx.org/[nginx] and https://ws.apache.org/[Apache Web Services](TM).
```

Comment:
No &tm; entity or such to properly denote a trademark?

Reply:
<https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#text-replacements>